File: platform/sdk/nodejs/examples/graph/graph-014_spark_stats_accumulate.js
const { GraphClient } = require("../../mox-sdk-graph");
const client = new GraphClient({ endpoint: "graph.local" });
const result = client.sparkStatsAccumulate();
if (!result.ok || !result.accumulated.totalTransactions) process.exit(1);
console.log("XJ-OK: graph-014_spark_stats_accumulate");
process.exit(0);

File: platform/sdk/python/examples/graph/graph-014_spark_stats_accumulate.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from mox_sdk_graph import GraphClient

client = GraphClient({"endpoint": "graph.local"})
result = client.sparkStatsAccumulate()
if not result["ok"] or not result["accumulated"]["totalTransactions"]:
    sys.exit(1)
print("XJ-OK: graph-014_spark_stats_accumulate")
sys.exit(0)
